.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 110,
+  "singleQuote": true,
+  "semi": false
```

Comment:
No please don't - I hate Prettier 😅 ESLint is sufficient for the formatting.

Reply:
LOL. I had a feeling I might get this reaction. TBH the only reason I brought this in was because I couldn't save files without conflicting with the standards. These changes were really just meant so that folks who have the prettier **vs-code** plugin in place can contribute without too much issue. I think the `singleQuote` and `semi` setting is just reflecting your current eslint style. 

Anyway, I can see that this may feel like "polluting" but it does help those "prettier folks" obey the style standards and contribute.

Reply:
I can remove these files once we're agreed on everything else

Reply:
We could add `prettier: false` to `.vscode/settings.json`

And please revert the format changes, thanks

Reply:
Good idea, let me try that now.

Reply:
I really dislike polluting other people's repos with files just so I can contribute (especially someone like yourself who's work I have so much respect for ... and I know you must be busy with a million things) but unless i'm somehow mistaken, there must be "default features" or "user features" that are able to permeate and make this repo non-idempotent across developers.

Reply:
Ok, i went into my User settings and found these lines which felt as though they might have conflicted in my initial checkout experience (not to mention I have no affinity to these settings and they even seem to conflict a bit).

<img width="348" alt="image" src="https://user-images.githubusercontent.com/980757/157269122-6076f088-c845-4a15-982c-4b2cad90c457.png">

after doing that and running **eslint** with the `--fix` flag on I believe all files are in their proper configuration.

Reply:
I did add this script target if you're ok with it but I'll remove this too if you don't want it:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/980757/157269856-107c1b4e-a577-4547-b974-31a00bbce3fb.png">